hack/pre-release.sh

```diff
@@ -187,9 +190,20 @@ switch_git_branch $base_branch $new_branch
 WMCO_VERSION=$(sed -n -e 's/^.*WMCO_VERSION ?= //p' Makefile)
 update_WMCO_version $stream $WMCO_VERSION
 update_OCP_version $stream $WMCO_VERSION 
+
+# check to see if the opeator-sdk version is correct 
+# if not, update it to the correct version 
+operator_sdk_current_version=$(operator-sdk version |  grep -oP 'operator-sdk version: "\K[^"]+' )
+if [[ "$operator_sdk_current_version" != "$OPERATOR_SDK_VERSION" ]]; then
```

Comment:
Does this account for the SDK not being installed also?

Reply:
if operator-sdk isn't installed at all, the command that would get the version would fail, and the script would exit. Should we handle this in a different way?

Reply:
I suggest making the error message more generic.